lib/openstax/aws/rds_instance.rb

```diff
@@ -0,0 +1,26 @@
+module OpenStax::Aws
+  class RdsInstance
+
+    attr_reader :raw
+
+    delegate_missing_to :@raw
+
+    def initialize(db_instance_identifier:, region:)
+      @raw = Aws::RDS::DBInstance.new(
+        db_instance_identifier,
+        client: Aws::RDS::Client.new(region: region)
+      )
+    end
+
+    def set_rds_master_password( db_instance_identifier: ,password:)
```

Comment:
Since this is already an RDS class, you can probably remove `_rds_` from the method name.  And per below, don't need the `db_instance_identifier`.